torch_patches/X10-device_test.diff

```diff
@@ -19,10 +19,10 @@ index a23696c29c..80e95f9a15 100644
                          self.precision = self._get_precision_override(test, dtype)
                          result = test(self, device_arg, dtype)
                      finally:
-@@ -242,10 +243,95 @@ class CUDATestBase(DeviceTypeTestBase):
+@@ -242,10 +243,102 @@ class CUDATestBase(DeviceTypeTestBase):
          cls.primary_device = 'cuda:{0}'.format(torch.cuda.current_device())
- 
- 
+
+
```

Comment:
Can we design this patch such that we can pass the path of a Python configuration module to the PT generic test infra, and they load it (and fetch the required code implementation and configuration)?
Would be nice to get out of having to carry a patch 😉

Reply:
I think so. I added it to the task tracker.